addons/main/functions/fnc_3denEventHandlers_onConnectionEnd.sqf

```diff
@@ -0,0 +1,112 @@
+// _type <STRING> = Connection Name/Type; same as data property
+// _from <[[OBJECT]]> = Array of 3den Objects starting the connection; You can multiselect and connect all at once
+// _to <OBJECT> = 3den Object the connection goes to
+// Array of Eden entities
+// [objects, groups, triggers, systems, waypoints, markers, layers, comments]
+// https://community.bistudio.com/wiki/Array_of_Eden_Entities
+
+params ["_type", "_from", "_to"];
+
+//(format ["_type: %1 - _from: %2 - _to: %3", _type, _from, _to]) call BIS_fnc_3DENNotification;
+
+private _allowedTypes = ["AE3_PowerConnection", "AE3_NetworkConnection"]; // These are the "data" values of the connections; in my case equal to class name
+
+// _to is nil if connection got canceled
+if ((isNil "_to") || !(_type in _allowedTypes)) exitWith {};
+
+/* ================================================================================ */
+
+// Event Handler currently not necessary
+/* _to addEventHandler 
+[
+    "ConnectionChanged3DEN",
+    {
+	    params ["_object"];
+
+        //(format ['_object: %1', _object]) call BIS_fnc_3DENNotification;
+    }
+]; */
+
+/* ================================================================================ */
+
+if (_type isEqualTo "AE3_NetworkConnection") then
+{
+    private _allowedNetworkFromClasses =
```

Comment:
In the future we should introduce a network config in the CfgVehicles of network devices, so we 
dont have to hard code the types here.

Reply:
Yes you are right. For the moment I will leave this as it is and create an issue for that.